frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.module.css

```diff
@@ -2,25 +2,48 @@ body {
   background-color: transparent;
 }
 
+.NoBackground {
+  --mb-color-embed-bg-color-override: transparent;
+}
+
+.ThemeNight.NoBackground {
+  --mb-color-embed-font-color-override: var(--mb-color-text-dark);
+}
+
 .EmbedFrame {
-  background-color: var(--mb-color-bg-dashboard);
+  background-color: var(
+    --mb-color-embed-bg-color-override,
+    var(--mb-color-bg-dashboard)
+  );
```

Comment:
@heypoom let me know if this looks ok. I'm still new to theming our part of the SDK. I don't know if there's an established solution for overriding CSS custom properties, but this is the solution I chose.

Reply:
@WiNloSt Looks okay to me! Since this is for iframe embedding and the CSS variables are statically defined in the CSS stylesheet, I think this is the only way possible at the moment.

Reply:
I was thinking about structuring the css custom properties another way e.g. don't use fallback values like this but define the color with anothre custom property e.g.
```
.noBackground {
--mb-color-my-color: transparent;
}

.some-rule {
--mb-color-my-color: default-color;
background-color: var(--mb-color-my-color);
}
```

This has a problem where I need to ensure the overriding `--mb-color-mycolor` has the correct specificity so I might need to add a bunch of block with `.noBackground` setting this `--mb-color-my-color: transparent`. That's why I think this is currently the best solution for now, since we only set the overriding color once, if it's not set then the fallback value would be used and we don't need to deal with the CSS specificity.